src/common/Input/Input.module.css

```
@@ -0,0 +1,20 @@
+input {
```

you were right the first time, sorry!
```suggestion
.input {
```